recordset/recordset.html

```diff
@@ -32,35 +32,15 @@ <h2 id="page-title">Records for {{ vm.tableDisplayName }}</h2>
                 <button class="btn btn-sm btn-default dropdown-toggle" type="button" data-toggle="dropdown">Items per page
                     <span class="caret"></span></button>
                 <ul class="dropdown-menu">
-                    <li><a href ng-click="pageLimit(10)">
-                        <span ng-show="pageInfo.pageLimit === 10" class="glyphicon glyphicon-ok"></span>
-                        <span ng-show="pageInfo.pageLimit !== 10" class="glyphicon glyphicon-invisible"></span>
-                        10</a>
+                    <li ng-if="pageLimits.indexOf(pageInfo.pageLimit) === -1"><a id="custom-page-size" href ng-click="pageLimit(pageInfo.pageLimit)">
+                        <span ng-show="true" class="glyphicon glyphicon-ok"></span>
```

Comment:
You don't need `ng-show` here because this element should only be drawn if the ng-if 1 line above passes.